appveyor.yml

```diff
@@ -11,12 +11,18 @@ init:
 
 install:
 
-- git clone --branch=master https://github.com/OneGet/oneget.git c:\projects\NuGetTest\OneGet
 - ps: $psversiontable
 - reg ADD "HKLM\Software\Microsoft\StrongName\Verification\Microsoft.PackageManagement.NuGetProvider,31bf3856ad364e35"  /f
 - reg ADD "HKLM\Software\Wow6432Node\Microsoft\StrongName\Verification\Microsoft.PackageManagement.NuGetProvider,31bf3856ad364e35"  /f
 
-
+before_build:
+  - ps: pushd c:\projects\NuGetTest\NuGetProvider
+  - ps: c:\projects\NuGetTest\NuGetProvider\Generate-Resources.ps1
+  - ps: $null = Invoke-WebRequest https://dist.nuget.org/win-x86-commandline/latest/nuget.exe -OutFile c:\projects\NuGetTest\NuGetProvider\nuget.exe
+  - ps: pushd c:\projects\NuGetTest\NuGetProvider\vs-csproj
+  - ps: c:\projects\NuGetTest\NuGetProvider\nuget.exe restore -PackagesDirectory c:\projects\NuGetTest\NuGetProvider\vs-csproj\packages
+  - ps: popd
+  - ps: popd
 build_script:
   - msbuild "c:\projects\NuGetTest\NuGetProvider\vs-csproj\NugetLightProvider.csproj" /verbosity:normal
 after_build:
```

Comment:
Should we update our smoke test cases to make sure we are using the latest nuget provider?

Reply:
Not sure what you mean. Can you elaborate / rephrase please?

Reply:
There are some tests under nugetprovider.